sdks/python/apache_beam/io/gcp/bigquery_read_internal.py

```diff
@@ -302,7 +303,7 @@ def _execute_query(
         self._job_name,
         self._source_uuid,
         bigquery_tools.BigQueryJobTypes.QUERY,
-        '%s_%s' % (int(time.time()), random.randint(0, 1000)))
+        '%s_%s' % (int(time.time()), secrets.token_hex(3)))
```

Comment:
in alignment with https://github.com/apache/beam/blob/0b61035f36fb099a8dcd39978c71779a2e81f957/sdks/python/apache_beam/io/gcp/bigquery.py#L2961

likely the cause of #26343